setup.py

```diff
@@ -3,53 +3,51 @@
 
 import os
 import codecs
-from setuptools import setup
+from setuptools import setup, find_packages
 
 
 def read(fname):
     file_path = os.path.join(os.path.dirname(__file__), fname)
-    return codecs.open(file_path, encoding='utf-8').read()
+    return codecs.open(file_path, encoding="utf-8").read()
 
 
 setup(
-    name='pytest-regressions',
-    version='1.0.0',
-    author='ESSS',
-    author_email='foss@esss.co',
-    maintainer='Bruno Oliveira',
-    maintainer_email='bruno@esss.co',
-    license='MIT',
-    url='https://github.com/ESSS/pytest-regressions',
-    description='Easy to use fixtures to write regression tests.',
-    long_description=read('README.rst'),
-    py_modules=['pytest_regressions'],
-    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
+    name="pytest-regressions",
+    version="1.0.0",
+    author="ESSS",
+    author_email="foss@esss.co",
+    maintainer="Bruno Oliveira",
+    maintainer_email="bruno@esss.co",
+    license="MIT",
+    url="https://github.com/ESSS/pytest-regressions",
+    description="Easy to use fixtures to write regression tests.",
+    long_description=read("README.rst"),
+    packages=find_packages("src"),
+    package_dir={"": "src"},
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*",
+    extras_require={"dev": ["pre-commit", "restructuredtext-lint"]},
```

Comment:
Is this installed using pyvenv?

Reply:
Using `pip`: `pip install -e .[dev]` for example